src/themes/teams-dark/components/Avatar/avatarStyles.ts

```diff
@@ -0,0 +1,40 @@
+import { pxToRem } from '../../../../lib'
```

Comment:
note that these styles are one-to-one the styles that are used for the default theme. To eliminate code repetition we could just reuse the ones that are used for default theme.

```js
const darkTheme = mergeThemes(defaultTheme, darkThemeOverrides)
```